vNext/src/Orleans/Orleans.csproj

```diff
@@ -33,6 +33,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="1.1.1" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="2.0.0-preview2-final" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.0.0-preview2-final" />
```

Comment:
Logging 2.0.0 was released. Although it's a .NET Standard 2.0 library, so let's figure this out before we even try to merge